src/Stratis.Bitcoin.Features.Wallet/OpReturnDataReader.cs

```diff
@@ -5,10 +5,9 @@
 using System.Text.RegularExpressions;
 using NBitcoin;
 using NLog;
-using Stratis.Features.Collateral.CounterChain;
 using TracerAttributes;
 
-namespace Stratis.Features.FederatedPeg
+namespace Stratis.Bitcoin.Features.Wallet
```

Comment:
Im thinking this might better places in `Stratis.Bitcoin` so we dont create more project dependencies? Perhaps in a folder under `Stratis.Bitcoin` called `Sidechain` or `CrossChain`? 

IMO it is a core class and generic enough to be put there rather. Thoughts?

Reply:
Done.